web_tool_script_3.R

```diff
@@ -20,24 +20,23 @@ set_portfolio_parameters(file_path = fs::path(par_file_path, paste0(portfolio_na
 analysis_inputs_path <- set_analysis_inputs_path(twodii_internal, data_location_ext, dataprep_timestamp)
 
 source(file.path(template_path, "create_interactive_report.R"))
+source(file.path(template_path, "create_executive_summary.R"))
+source(file.path(template_path, "useful_functions.R"))
 
+repo_path <- template_path
+template_name <- paste0("template_", tolower(language_select))
+template_dir <- fs::path(template_path, template_name)
+template_dir <- paste0(repo_path,"swiss_en_template/_book/")
```

Comment:
this seems odd. define the same path twice in a row with different paths?? @Clare2D

Reply:
@cjyetman @maurolepore

Reply:
agreed... L28 seems totally pointless

Reply:
Still getting there on finalising how this will be selected. Will update soon

Reply:
@Clare2D I think this still needs to be resolved before we merge

Reply:
This is updated in a new PR.

Reply:
do you mean PR #226? does #226 supersede this PR, or are the two PRs mutually dependent?

Reply:
I kept on working. In creating new PRs it seemed to take the old code and all the commits and ran from there. If we can get this PR merged, then this will hopefully be resolved. I could have just added the new changes to this PR but I thought that would confuse further.